src/sql/sqlops.d.ts

```diff
@@ -1061,13 +1061,13 @@ declare module 'sqlops' {
 	}
 
 	export interface AgentJobHistoryInfo {
-		instanceID: number;
+		instanceId: number;
```

Comment:
any reason we're switching ID to Id?  In Typescript is the normal convention `ID`?  In any case, I don't think it's a good idea to mix and match the two casing conventions within a single interface.

I think I may have been using `Id` since that is the C# convention.  If you see that, please feel free to switch to the correct convention or note in code review.  Thanks!

Reply:
@kburtram Every single variable or field with `ID` in the `sqlops.d.ts` file is in Pascal case (`Id`). I'm assuming it's because that's the way we receive it from the service.